templates/main/base.html

```diff
@@ -42,7 +42,7 @@
                     <p>
                         This website uses <a href="https://python.org">Python</a> and
                         <a href="http://flask.pocoo.org/">Flask</a>, and was developed collaboratively on
-                        <a href="https://gitlab.com/python-discord/projects/site">GitLab</a>.
+                        <a href="https://github.com/python-discord/projects/site">GitLab</a>.
```

Comment:
The text should probably be changed here.

Reply:
This URL is also incorrect, it is https://github.com/python-discord/site not https://github.com/python-discord/projects/site

Reply:
```suggestion
                        <a href="https://github.com/python-discord/site">GitHub</a>.
```